PowerPointLabs/PowerPointLabs/Utils/ClipboardUtil.cs

```diff
@@ -233,5 +200,38 @@ private static Shape TryPastingOntoView(PowerPointPresentation pres, PowerPointS
                 return null;
             }
         }
+
+        private static ShapeRange PasteWithCorrectSlideCheck(PowerPointSlide slide, bool isPasteSpecial = false, PpPasteDataType pasteType = PpPasteDataType.ppPasteDefault)
```

Comment:
Just wondering, was there any particular reason why you shifted this method downwards? It will conflict with my other PR haha.